packages/integrations/markdoc/README.md

```diff
@@ -458,6 +458,39 @@ To achieve a more Markdown-like experience, where HTML elements can be included
 > **Warning**
 > When `allowHTML` is enabled, HTML markup inside Markdoc documents will be rendered as actual HTML elements (including `<script>`), making attack vectors like XSS possible. Ensure that any HTML markup comes from trusted sources.
 
+### `allowIndentation`
+
+Enables indentation of Markdoc tags for better readability.
+
+The allowIndentation option is experimental and is not enabled by default. When the option is enabled, it is not possible to use indent-based code blocks.
```

Comment:
I don't think a call out saying it's experimental is necessary. Both this integration and markdoc itself are pre-1.0, and we are just exposing one of its config options that doesn't seem to be any more experimental than the other features.